hipcub/include/hipcub/backend/rocprim/device/device_scan.hpp

```diff
@@ -95,16 +100,16 @@ class DeviceScan
                             size_t &temp_storage_bytes,
                             InputIteratorT d_in,
                             OutputIteratorT d_out,
-                            int num_items,
+                            size_t num_items,
                             hipStream_t stream = 0,
-                            bool debug_synchronous = false,
-                            size_t size_limit = size_t(std::numeric_limits<int>::max()) + 1)
+                            bool debug_synchronous = false)
     {
         using T = typename std::iterator_traits<InputIteratorT>::value_type;
         return ExclusiveScan(
             d_temp_storage, temp_storage_bytes,
             d_in, d_out, ::hipcub::Sum(), T(0), num_items,
-            stream, debug_synchronous, size_limit
+            stream, debug_synchronous,
+            HIPCUB_DEVICE_SCAN_SIZE_LIMIT
```

Comment:
Same here, can't pass HIPCUB_DEVICE_SCAN_SIZE_LIMIT since its no longer a parameter in ExclusiveScan